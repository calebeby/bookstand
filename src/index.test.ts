import { createRollupConfig, start } from '.'

// const groups = {
//   groups: [
//     {
//       name: 'Styles',
//       patterns: [
//         {
//           name: 'Fonts',
//           id: 'lksadjflskjdfs',
//           renderer: 'bookstand-html-renderer',
//           rendererOpts: {},
//           patternPath: './patterns/fonts',
//         },
//       ],
//     },
//     {
//       name: 'Components',
//       patterns: [
//         {
//           name: 'Button',
//           id: 'aslkdfjasdlkfj',
//           renderer: 'bookstand-preact-renderer',
//           rendererOpts: {
//             opts: 'here',
//           },
//           patternPath: './components/button',
//         },
//         {
//           name: 'TextInput',
//           id: 'kfjsdljlkaslfa',
//           renderer: 'bookstand-preact-renderer',
//           rendererOpts: {},
//           patternPath: './components/text-input',
//         },
//       ],
//     },
//   ],
// }

describe('createRollupConfig', () => {
  it('should match snapshot', () => {
    expect(createRollupConfig('')).toMatchSnapshot()
  })
})

describe('start', () => {
  it('should create a bundle', () => {
    start()
  })
})
