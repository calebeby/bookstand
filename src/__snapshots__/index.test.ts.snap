// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createRollupConfig should match snapshot 1`] = `
Object {
  "input": "virtual-module",
  "output": Object {
    "file": "dist/bundle.js",
    "format": "iife",
    "name": "Bundle",
  },
  "plugins": Array [
    Object {
      "load": [Function],
      "name": "rollup-plugin-virtual-resolver",
      "resolveId": [Function],
    },
  ],
}
`;
